internal/pkg/levee/testdata/src/example.com/tests/callorder/beforesource.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package callorder
+
+import (
+	"example.com/core"
+)
+
+type key struct {
+	name string
+}
+
```

Comment:
Even with the context of this PR's `Conversation`, I was caught off guard here.  I seemed like the tests were expecting this to be identified as a source.

Consider adding a comment to the `type key` that we do *not* expect this to be a source, nor do we expect taint to through its use below.

Reply:
SG. I've added a comment to the `type` declaration that it shouldn't be identified as a source.

For the `do not expect taint` part I added comments in each test, in case we add other cases later were we *do* expect the `key` to be tainted. (Also I think having the comments in the cases makes it easier to understand the intent when reading just the test cases).